client/js/components/month/month-day-card.js

```diff
@@ -1,31 +1,25 @@
 /** @jsx React.DOM */
 var React = require('react');
+var MonthDayEvent = require('./month-day-event');
 var Link = require('react-router-component').Link;
+var moment = require('moment');
 
 var MonthDayCard = React.createClass({
   render:function () {
-    var currentDate = this.props.dayData.date;
-    var dayPath = '/day/' + currentDate;
+    var weekNumber = Number(this.props.dayData.weekNumber);
```

Comment:
Good catch, it was from an old version where I needed to pass in the week and day and do a calculation based on that